deploy/ray-operator/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM golang:1.12.5 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY main.go main.go
+COPY api/ api/
+COPY controllers/ controllers/
```

Comment:
Can we copy the whole directory into the builder? It has too many layers.

Can we avoid go mod instruction?  It takes much more time.

If you think the whole directory is large, we can use dockerignore.

Reply:
Yes , I am working on optimizing this part.